### ISSUE TITLE
Noto Emoji (Monochrome) version

### ISSUE THREAD

**Issue by Symbian9** [efck-chat-keyboard/efck#12]
**Is your feature request related to a problem? Please describe.**

*Noto Color Emoji* is huge.

**Describe the solution you'd like**

Make a version with *Noto Emoji* (Monochrome) dependency, instead of *Noto Color Emoji*.

- https://fonts.google.com/noto/specimen/Noto+Emoji

**Additional context**

Make use static `NotoEmoji-Regular.ttf` (not variable `NotoEmoji-VariableFont_wght.ttf`).

- ZIP: https://fonts.google.com/download?family=Noto%20Emoji

**Comment by kernc** [comment]
We _don't_ bundle the font for Windows (not needed / doesn't work), macOS (not needed / doesn't work), or Debian (font is installed via a package dependency). We only bundle the font for the linux tgz archive.

The supported icon fonts are listed here:
https://github.com/efck-chat-keyboard/efck/blob/096a7b6cc4004911c06337fa736a25ac67e1e628/efck/tabs/emoji.py#L100-L141
I guess we could add `"Noto Emoji"` in there somewhere ...

Also note, you can [force any emoji font with `ICON_FONT` environment variable](https://efck-chat-keyboard.github.io/faq/#custom-font).

**Comment by Symbian9** [comment]
> or Debian (font is installed via a package dependency)

Yes, above I mean DEB strong dependency on `fonts-noto-color-emoji` — this package is outdated in all stable distros:

- https://repology.org/project/fonts:noto-emoji/versions

So, it might be good set it to `Recommends` (instead of `Depends`) in `../debian/control`.

**Comment by kernc** [comment]
But if we only recommend it, then we have to vendor a fallback font, which can be "Noto Emoji", but that would increase the DEB package size by 700%! Any other ideas?

What's wrong with the outdated font besides a few missing glyphs? :sweat_smile:
